script/release-workflow/git-push.sh

```diff
@@ -2,6 +2,10 @@
 
 set -euo >/dev/null
 
+if [[ $(git status --porcelain) ]]; then
+    git add Gemfile.lock VERSION lib/pact/cli/version.rb
+    git commit -m "chore(release): version ${TAG}"
+fi
```

Comment:
This doesn't feel like it should be in here. There should be another file that does the commit.